### PR TITLE
Fix crash when try to access inavlid ZEPlayer*

### PR DESCRIPTION
**Pull request by STONE-GPY**
It accesses a nullptr and cause crashes.

You can probably make it happen by using the kick and mute commands simultaneously.

### CODE REVIEW

**Comment by Poggicek** [comment]
TargetPlayerString shouldn't return invalid players in the first place

**Comment by STONE-GPY** [comment]
But I get crash in that place today.
I checked server access to pPlayer->IsFakeClient() when pPlayer is nullptr that on mute commands at minidump. 
There are moments the player controller is valid and ZEPlayer is not valid.

**Comment by Poggicek** [comment]
No I'm not saying that it's not possible. I'm saying that the fix should be inside TargetPlayerString. Not around it

**Comment by STONE-GPY** [comment]
I understand now, just add a check player connected at TargetPlayerString.

File: src/playermanager.cpp [hunk 1]
```diff
@@ -376,7 +376,9 @@ ETargetType CPlayerManager::TargetPlayerString(int iCommandClient, const char* t
 		if (userid != -1)
 		{
 			targetType = ETargetType::PLAYER;
-			clients[iNumClients++] = GetSlotFromUserId(userid).Get();
+			CCSPlayerController* player = CCSPlayerController::FromSlot(GetSlotFromUserId(userid).Get());
+			if(player && player->IsController() && player->IsConnected())
+				clients[iNumClients++] = GetSlotFromUserId(userid).Get();
```
Comment (xen-000) [inline src/playermanager.cpp:381]: This is only for userid targeting, you should do this in all the other cases as well.

Comment (STONE-GPY) [reply src/playermanager.cpp:386]: okay, i add it to other cases.

**Review by xen-000 — APPROVED** [review]

**Pull request merged**